ACKNOWLEDGMENTS.md

```diff
@@ -20,4 +20,4 @@ Julian Berro, Jiongyi Tan, Margot Quinlan, Lam Nguyen, Tamara Bidone, Glen Hocky
 Ludovic Autin and Ivan Viola
 
 ## For inspiration and/or comments on manuscripts:
-Rick Horwitz, Julie Theriot
```

Comment:
Not sure why this is shown as a diff, but it's the same. Google says "acknowledgements" is British and "acknowledgments" is American, so I changed the file name and use the American spelling elsewhere too. Let me know if this is an issue for some reason.